### ISSUE TITLE
platform check

### ISSUE THREAD

**Issue by aremazeilles** [aremazeilles/beat_routine#12]
In several files, you check the protocol from plaformData file

If incorrect, you do a print, like:

```shell
./run_emg tests/data/input/subject_01_run_01_emg.csv tests/data/input/subject_01_run_01_PlatformData_protocol3.csv out_tests               1 ↵
current directory: /home/anthony/code/eurobench/beat_routine
csv file: tests/data/input/subject_01_run_01_emg.csv
platform data: tests/data/input/subject_01_run_01_PlatformData_protocol3.csv
Same sample frequencyYou have tried to lunch EMG_routine with a wrong protocolwarning: EMG_routine: some elements in list of return values are undefined
warning: called from
    EMG_routine at line 70 column 3
    ./run_emg at line 19 column 5
NoS = [](0x0)
╭─anthony@PRT0
```

When you detect the file is not conform, it would be better to return directly, and not try continue the process.
On my machine the prints does not have a return at the end of a sentence. Is is lso the case for you? You may have to include a ' \n' at the end of your string

**Comment by juritaborri** [comment]
I have inserted return in each script when protocol check fails. And I also added the sentence related to the accepted protocol.

**Comment by aremazeilles** [comment]
I just made a commit to your branch "Protocol Check". 
Once you are happy with your changes, from the web interface, you have to:
*    Go to you r branch (Like code->branches)
* Create a New Pull request, 
* In the text at least write Fix #12 (this will close this issues once the Pull Request is processed)
* Assign it to me (on the right side)

Do you want to try it?

**Comment by juritaborri** [comment]
Let me know if the two pull requests are ok
